manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -275,10 +275,19 @@ spec:
           secretName: istiod-tls
           optional: true
       - name: istio-csr-ca-configmap
+  {{- if (.Values.env).ENABLE_CLUSTER_TRUST_BUNDLE_API }}
+        projected:
+          sources:
+          - clusterTrustBundle:
+              name: istio-ca-root-cert
```

Comment:
I am not sure the answer to this: are there plausible use cases where a user may deploy a cert + a trust bundle, and Istio just consumes that and doesn't create it?

Seems like 'maybe' and we can add that later without issue.

Reply:
I agree with 'maybe' being the answer here. For now, this only covers the simplest use cases where a) the internal CA is used with a self-signed cert generated by istiod or b) with a cert passed into istiod through the cacerts secret. In both cases, istiod will manage the ClusterTrustBundle. We might want to keep it that way to keep things simple - ie if you use an external CA to manage the cert/key, that only has to update the cacerts secret and not bother with ClusterTrustBundle